astra/src/main/java/com/slack/astra/metadata/schema/ChunkSchema.java

```diff
@@ -24,7 +26,7 @@ public static void serializeToFile(ChunkSchema chunkSchema, File file) throws IO
   }
 
   public static ChunkSchema deserializeBytes(byte[] bytes) throws IOException {
-    return serDe.fromJsonStr(new String(bytes));
+    return serDe.fromJsonStr(new String(bytes, UTF_8));
```

Comment:
Let's make sure this change is backwards compatible. Ensure no serde warnings when we deploy.